bin/create

```diff
@@ -14,36 +15,36 @@ if [ $(id -u) -ne 0 ]; then
     exit 1
 fi
 
-CORE=""
-if [ -e /snap/core/current ]; then
-    CORE="core"
-elif [ -e /snap/ubuntu-core/current ]; then
-    CORE="ubuntu-core"
-else
-    echo "Cannot find core snap"
+# FIXME: We really should ensure this also works on
+#        classic, its an easy way to get a chroot there
+if ! grep -q "Ubuntu Core" /etc/os-release; then
+    echo "Classic mode only works on Ubuntu Core"
     exit 1
 fi
 
-# FIXME: confinement will prevent this
 echo "Creating classic environment"
-VERSION="$(/bin/readlink /snap/${CORE}/current)"
-SNAPROOT="/var/lib/snapd/snaps/${CORE}_${VERSION}.snap"
-/usr/bin/unsquashfs -d $ROOT $SNAPROOT
-mkdir $ROOT/snappy
+mkdir -p $ROOT
+tar -x -f $SNAP/bionic-base-*.tar.gz -C $ROOT
```

Comment:
Nitpick: I'd feel better if we could do it without a glob. We ship $SNAP so we should be able to provide a consistent location.